catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/receive/NotificationMessageReceiverTest.scala

```diff
@@ -195,40 +167,32 @@ class NotificationMessageReceiverTest
   }
 
   it("fails if it's unable to publish the work") {
-    val sierraTransformable = SierraTransformable(
-      bibRecord = createSierraBibRecord
-    )
+    withLocalSqsQueue { _ =>
+      withLocalS3Bucket { bucket =>
+        val message = createMessageWith(bucket = bucket)
 
-    withLocalSnsTopic { topic =>
-      withLocalSqsQueue { _ =>
-        withLocalS3Bucket { bucket =>
-          val message = hybridRecordNotificationMessage(
-            message = JsonUtil.toJson(sierraTransformable).get,
-            sourceName = "sierra",
-            version = 1,
-            s3Client = s3Client,
-            bucket = bucket
-          )
-
-          withNotificationMessageReceiver(
-            topic,
-            bucket,
-            Some(mockSnsClientFailPublishMessage)) { recordReceiver =>
+        withNotificationMessageReceiver(Topic("does-not-exist"), bucket) {
+          recordReceiver =>
             val future = recordReceiver.receiveMessage(message)
 
             whenReady(future.failed) { x =>
-              x.getMessage should be("Failed publishing message")
+              x.getMessage.startsWith("Unknown topic: does-not-exist") shouldBe true
             }
```

Comment:
Ah, so I tried this when I first wrote the code and the compiler got miserable because `startWith` expected a Word and I couldn't work out how to make it work after three attempts

Reply:
Ah when it says that it's usually about parenthesis. Try:
```
x.getMessage should startWith ("Unknown topic: does-not-exist")
```